src/lib/storage/extensions.rb

```diff
@@ -20,26 +20,21 @@
 # find current contact information at www.suse.com.
 
 require "storage"
-require "expert-partitioner/icons"
+require "expert_partitioner/icons"
 
 include Yast::UIShortcuts
 
-
 module Storage
-
-
   def self.silence
-    silencer = ::Storage::Silencer.new()
+    silencer = ::Storage::Silencer.new
     yield
   ensure
     silencer.turn_off
   end
 
-
   class Device
-
+    # rubocop:disable Metrics/CyclomaticComplexity
```

Comment:
I disagree it do
1) create Header term
2) remap fields from ids to its name
3) align Size element to right ( I do not get why, some comment would be helpful )

BTW it also have so bit cyclomatic complexity because it do mapping inside of Header call, which is also quite discouraged to have multiline map inside method call arguments. I think even simple
```
names = fields.map do |field|
  ...
end

return Header(*names)
```
will help with cyclomatic complain.